tests/unit/expression_test.py

```diff
@@ -671,6 +677,9 @@ def test_mod_variables() -> None:
 
 def test_mod_z3expr() -> None:
     assert Mod(Number(1000), Number(5)).z3expr() == z3.IntVal(1000) % z3.IntVal(5)
+    x = Mod(Number(1000), Number(5))
+    assert x.z3expr() == z3.IntVal(1000) % z3.IntVal(5)
+    assert x.z3expr() == z3.IntVal(1000) % z3.IntVal(5)
```

Comment:
Why is this test case only added to `Not`, `Mod` and `Div`? Wouldn't it be better to test that in a generic unit test function (or if that is not possible, for all expressions)?

Reply:
Obsolete as LRU caching is used now (cf. 83083cf).